components/automate-ui/src/app/modules/infra-proxy/policy-files/policy-files.component.html

```diff
@@ -4,9 +4,9 @@
   <ng-container *ngIf="!policyFilesListLoading && !authFailure">
     <chef-table>
       <chef-thead>
-        <chef-tr>
-          <chef-th>Policy Name</chef-th>
```

Comment:
Neither your PR preamble nor the associated issue mentions this policy page--so is this really supposed to be included?

Reply:
yes for now.